### PR TITLE
[Relay][TOPI][OP] intel_graphics conv2d alterlayout support relay, added stack op

### PR DESCRIPTION
**Pull request by Laurawly**
Also concatenate op moved compute to align with other ops.

### CODE REVIEW

File: topi/python/topi/intel_graphics/conv2d.py [hunk 1]
```diff
@@ -64,7 +59,12 @@ def _alter_conv2d_layout(attrs, inputs, tinfos, F):
     new_attrs = {k: attrs[k] for k in attrs.keys()}
     new_attrs['kernel_layout'] = 'OIHW%do' % (oc_bn)
 
-    return sym.contrib.conv2d_NCHWc(*copy_inputs, **new_attrs)
+    if F == sym:
+        out = F.contrib.conv2d_NCHWc(*copy_inputs, **new_attrs)
+    else:
+        out = F.nn.contrib_conv2d_nchwc(*copy_inputs, **new_attrs)
```
Comment (yzhliu) [inline topi/python/topi/intel_graphics/conv2d.py]: does it work? `Conv2dAttr` contains `layout` in nnvm while in Relay the name is `data_layout` though

Comment (Laurawly) [reply topi/python/topi/intel_graphics/conv2d.py]: It seems working for me.  `relay.nn.contrib_conv2d_nchwc`

Comment (yzhliu) [reply topi/python/topi/intel_graphics/conv2d.py]: oh yes, because it is copy, relay copies to relay, nnvm copies to nnvm.

**Comment by yzhliu** [comment]
Thanks @Laurawly

**Pull request merged**